packages/platforms/inventec/x86-64/x86-64-inventec-d7054q28b/modules/builds/inv_eeprom.c

```diff
@@ -0,0 +1,193 @@
+#include <linux/kernel.h>
```

Comment:
I'm assuming this custom module is necessary for your platform (as opposed to the normal eeprom drivers already present in the kernel). 

Is this particular module identical to the one for the 32q28b? If so you should move it to the inventec vendor module area and share it rather than duplicated it within each platform's module build.